README.md

```diff
@@ -19,7 +19,7 @@ Expected table:
 ![Result](./src/images/result.png)
 
 In this task, we don't provide details instructions. But below are some hints how to do this task:
-- JavaScript document has methods to work with tables. [Theory tables](https://javascript.info/dom-navigation#dom-navigation-tables) 
+- JavaScript document has methods to work with tables. [Theory tables](https://javascript.info/dom-navigation#dom-navigation-tables)
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.